project-sandbox/variables.tf

```diff
@@ -16,7 +16,7 @@ variable "project_name" {
 variable "apis_to_enable" {
   type        = list(string)
   description = "The list of additional APIs to enable. We always enable dataproc and cloudfunctions."
-  default     = []
+  default     = ["artifactregistry.googleapis.com"]
```

Comment:
`seqr` doesn't support `hail-batch` at the moment, so this perm isn't necessary.